Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.con
 ARG APT_MIRROR
 RUN sed -ri "s/(httpredir|deb).debian.org/${APT_MIRROR:-deb.debian.org}/g" /etc/apt/sources.list \
  && sed -ri "s/(security).debian.org/${APT_MIRROR:-security.debian.org}/g" /etc/apt/sources.list
+COPY /hack/dockerfile/certs/*.crt /usr/local/share/ca-certificates/
```

Comment:
Why not use `RUN --mount=type=secret`?

Reply:
Good idea, I didn't know about that feature. Since these are public certificates and not secret, I switched to `type=bind`, but could use `type=secret` if that has other benefits.